libmattermost.c

```diff
@@ -1213,7 +1248,8 @@ mm_fetch_url(MattermostAccount *ma, const gchar *url, const gchar *postdata, Mat
 	purple_debug_info("mattermost", "Fetching url %s\n", url);
 
 #if PURPLE_VERSION_CHECK(3, 0, 0)
-	
+	//FIXME: is this HTTP/1.0 ? with HTTP/1.1 Mattermost server v5 sends chunked data ..

```

Comment:
It is 1.1 yes.  The http lib handles chunked data too.

Reply:
OK, removed that comment.. with libpurple 2 and MM server sending chunked responses I've got some problems (but maybe due to my ancient^w stable rhel 7 ...)